tests/core/pyspec/eth2spec/test/helpers/execution_payload.py

```diff
@@ -263,6 +267,69 @@ def compute_el_block_hash_for_block(spec, block):
         spec, block.body.execution_payload, block.parent_root, requests_hash)
 
 
+def build_empty_execution_payload_header(spec, state):
+    latest = state.latest_execution_payload_header
+    timestamp = spec.compute_timestamp_at_slot(state, state.slot)
+    execution_payload = build_empty_execution_payload(spec, state)
+
+    execution_payload_header = spec.ExecutionPayloadHeader(
+        parent_hash = latest.block_hash,
```

Comment:
@jtraglia should `parent_hash` be just random Hash32 value here?

Reply:
This I'm not totally sure of, but I think a random value should be okay.